language-design.md

```diff
@@ -116,6 +314,11 @@ Briefly describe how you split up the work for this assignment.
 
 **Response**
 
-
+First, we did the readings separately. Then, we got together and talked about
+each of the questions in turn. We came up with parts of the texts we wanted to
+cite, and drafted a few points for each question together. After that, we
+agreed on a split for writing up questions and wrote up 3.5 answers each
+individually. Finally, we got back together, talked again about a few issues,
+and now we're done!
 
```

Comment:
Adam your critique was good too! I liked how you brought in stuff from class, and other languages (C, Python, Excel, Wolfram Alpha, microwaves, etc) and asked deeper questions about the responses.